Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -521,11 +521,10 @@ class BrowserViewController: UIViewController {
         super.viewDidAppear(animated)
         log.debug("BVC done.")
 
-        let appVersion = AppInfo.appVersion
-        if profile.prefs.stringForKey("latestAppVersion") != appVersion && DeviceInfo.hasConnectivity() {
+        if profile.prefs.stringForKey(LatestAppVersionProfileKey) != AppInfo.appVersion && DeviceInfo.hasConnectivity() {
```

Comment:
I think the logic here should be: *If this is an upgrade, (if the profile already existed), AND if `LatestAppVersionProfileKey` is not equal to `AppInfo.appVersion`, then show the What's New.

The check if the profile already exists is needed because we did not set `LatestAppVersionProfileKey` in `1.4`. That means that we need to look at some other profile preference to determine that this is not a clean install. What is a good preference that we always set on first run?